### PR TITLE
Including Geometry/HGCalMapping in categories_map

### PR DESCRIPTION
**Pull request by kerstinlovisa**
Including Geometry/HGCalMapping in categories_map.py under "geometry"

### CODE REVIEW

**Comment by iarspider** [comment]
assign geometry

**Comment by pfs** [comment]
@kerstinlovisa @all  i would suggest to add to `upgrades` as well for the moment

**Comment by iarspider** [comment]
assign geometry, upgrades

**Comment by kerstinlovisa** [comment]
@pfs @ALL HGCalMapping also added to upgrades category now

File: categories_map.py [hunk 1]
```diff
@@ -1632,6 +1633,7 @@
     "upgrade": [
         "CalibTracker/SiPhase2TrackerESProducers",
         "CondFormats/HGCalObjects",
+        "Geometry/HGCalMapping",
```
Comment (makortel) [inline categories_map.py:1636]: Please move this downwards to keep the alphabetic order.

Comment (kerstinlovisa) [reply categories_map.py]: Moved it down so it should be in alphabetic order now

**Comment by smuzaffar** [comment]
any objections @cms-sw/geometry-l2 @cms-sw/upgrade-l2  on adding `Geometry/HGCalMapping` under your control?

**Comment by civanch** [comment]
+1

**Pull request merged**